### PR TITLE
CI: Use runc v1.0.0-rc93 on hosts with archlinux containers

### PR DESCRIPTION
**Pull request by knzivid**
Can be reverted after these issues are fixed.

https://github.com/actions/virtual-environments/issues/2658
https://github.com/moby/moby/pull/42014

### CODE REVIEW

**Comment by eskaur** [comment]
Excellent. I can fix the "required tests" list so it's possible to merge.

**Comment by knzivid** [comment]
It's already green

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -14,13 +14,19 @@ jobs:
         uses: actions/checkout@v2
         with:
           lfs: false
+      - name: build latest runc # https://github.com/actions/virtual-environments/issues/2698
```
Comment (eskaur) [inline .github/workflows/main.yml:17]: You are referring to a different issue here than in the commit message. Was that intentional?

**Comment by eskaur** [comment]
> It's already green

Yes, because I removed "archlinux/base" as required check. It was stuck on Yellow.

File: .github/workflows/main.yml (hunk 1 above)
Comment (eskaur) [inline .github/workflows/main.yml:17]: Can you explicitly write "Remove step when {...} is solved" on both instances of "build latest runc" here?

Comment (knzivid) [reply .github/workflows/main.yml:17]: The issues in the commit message point to this issue's comment as a workaround.

Comment (knzivid) [reply .github/workflows/main.yml:17]: I created #122 to follow up on this PR. I can add a TODO(#122) in both places

Comment (eskaur) [reply .github/workflows/main.yml:17]: Ah yes, I should have looked more closely. :)

Comment (knzivid) [reply .github/workflows/main.yml]: I updated the yaml comment as well as the commit message to say this explicitly. Hope its better now :)

Comment (eskaur) [reply .github/workflows/main.yml]:
```
# TODO(#122): Remove step when PR #121 is solved
```

Don't you mean `# TODO(#122): Remove step when Issue #122 is solved`

Comment (knzivid) [reply .github/workflows/main.yml]: oh yeah, will fix

**Review by eskaur — APPROVED** [review]
Thanks for dealing with this. I'll remember to follow up on #122.

**Pull request merged**